Subject: [PATCH] Fix option for master wait
References: bsc#1178006

master wait expects a value, and if provided automount crashes

#0  __GI_____strtoul_l_internal (nptr=0x0, endptr=0x7fffffffe120, base=0, group=<optimized out>,
    loc=0x7ffff77a63a0 <_nl_global_locale>) at ../stdlib/strtol_l.c:292
#1  0x0000555555562c52 in getnumopt ()
#2  0x0000555555564ec0 in main ()

This is because the options string is not correct and does not expect
an argument for master wait (M)

Signed-off-by: Goldwyn Rodrigues <rgoldwyn@suse.com>

diff --git a/daemon/automount.c b/daemon/automount.c
index 0391bfb..c2a0f58 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -2219,7 +2219,7 @@ int main(int argc, char *argv[])
 	time_t timeout;
 	time_t age = monotonic_time(NULL);
 	struct rlimit rlim;
-	const char *options = "+hp:t:vmdD:SfVrO:l:n:CFM";
+	const char *options = "+hp:t:vmdD:SfVrO:l:n:CFM:";
 	static const struct option long_options[] = {
 		{"help", 0, 0, 'h'},
 		{"pid-file", 1, 0, 'p'},
